provider/k8s/app.go

```diff
@@ -272,14 +272,17 @@ func (p *Provider) appNameValidate(name string) error {
 func (p *Provider) appParametersUpdate(a *structs.App, params map[string]string) error {
 	defs := p.Engine.AppParameters()
 
+	var invalidParameters []string
 	for k, v := range params {
 		if _, ok := defs[k]; !ok {
-			return errors.WithStack(fmt.Errorf("invalid parameter: %s", k))
+			invalidParameters = append(invalidParameters, k)
+		} else {
+			a.Parameters[k] = v
 		}
-
-		a.Parameters[k] = v
 	}
 
+	fmt.Printf("Skipping invalid parameters: %s ...", strings.Join(invalidParameters, ", "))
```

Comment:
This message is a little scary. Maybe just remove the print entirely since there are no parameters on v3? If you keep it, make it something other than "invalid" which sounds like they did something wrong.